packages/primer-forms/lib/form-validation.scss

```diff
@@ -0,0 +1,398 @@
+// Needs refactoring
+// stylelint-disable selector-no-qualifying-type, selector-no-type
+dl.form-group > dd {
+  .form-control {
+    &.is-autocheck-loading,
+    &.is-autocheck-successful,
+    &.is-autocheck-errored {
+      padding-right: 30px; // Leave some space for our validation icons
+    }
+
+    &.is-autocheck-loading {
+      background-image: url("/images/spinners/octocat-spinner-16px.gif");
+    }
+
+    &.is-autocheck-successful {
+      background-image: url("/images/modules/ajax/success.png");
+    }
+
+    &.is-autocheck-errored {
+      background-image: url("/images/modules/ajax/error.png");
+    }
+  }
+}
+
+// Retinization of form validation icons that aren't octicons yet
+@include retina-media-query {
+  dl.form-group > dd {
+    .form-control {
+      &.is-autocheck-loading,
+      &.is-autocheck-successful,
+      &.is-autocheck-errored {
+        background-size: 16px 16px;
+      }
+
+      &.is-autocheck-loading {
+        background-image: url("/images/spinners/octocat-spinner-32.gif");
+      }
+
+      &.is-autocheck-successful {
+        background-image: url("/images/modules/ajax/success@2x.png");
+      }
+
+      &.is-autocheck-errored {
+        background-image: url("/images/modules/ajax/error@2x.png");
+      }
+    }
+  }
+}
+
+// Credit cards
+//
+// A selector for credit cards. Shows all credit cards we have available and
+// dims the non-selected ones.
+
+.form-cards {
+  height: 31px;
+  margin: 0 0 15px;
+
+  .card {
+    float: left;
+    width: 47px;
+    height: 31px;
+    text-indent: -9999px;
+    background-image: url("/images/modules/pricing/credit-cards-@1x.png");
+    background-position: 0 0;
+    opacity: 0.6;
+
+    &.visa { background-position: 0 0; }
+
+    &.amex { background-position: -50px 0; }
+
+    &.mastercard { background-position: -100px 0; }
+
+    &.discover { background-position: -150px 0; }
+
+    &.jcb { background-position: -200px 0; }
+
+    &.dinersclub { background-position: -250px 0; }
+
+    &.enabled { opacity: 1; }
+
+    &.disabled { opacity: 0.2; }
+  }
+
+  > .cards {
+    margin: 0;
+
+    > li {
+      float: left;
+      margin: 0 4px 0 0;
+      list-style-type: none;
+
+      &.text {
+        line-height: 31px;
+      }
+    }
+  }
+}
+
+@include retina-media-query {
+  .form-cards > .cards .card {
+    background-image: url("/images/modules/pricing/credit-cards-@2x.png");
+    background-size: 300px 31px;
+  }
+}
+
+// Inline verification
+// This overrides primer's inline form stuff
+.status-indicator {
+  display: inline-block;
+  width: 16px;
+  height: 16px;
+  margin-left: 5px;
+
+  .octicon {
+    display: none;
+  }
+}
+
```

Comment:
this is missing this line https://github.com/github/github/blob/master/app/assets/stylesheets/primer-core/forms/lib/form-validation.scss#L119-L122

```
.repo-config-option:not(:checked) ~ .time-remaining {
  display: none;
}
```